detection-rules/link_microsoft_low_reputation.yml

```diff
@@ -144,26 +149,29 @@ source: |
          )
      )
    )
- or (
-   any(ml.nlu_classifier(body.current_thread.text).intents,
-       .name == "cred_theft" and .confidence in~ ("medium", "high")
-   )
-   or any(attachments,
-          .file_type in $file_types_images
-          and any(file.explode(.),
-                  any(ml.nlu_classifier(.scan.ocr.raw).intents,
-                      .name == "cred_theft" and .confidence in ("medium", "high")
-                  )
-          )
-   )
    or (
-     any(ml.nlu_classifier(body.current_thread.text).entities, .name == "urgency")
-     and not any(ml.nlu_classifier(body.current_thread.text).intents,
-                 .name == "benign" and .confidence == "high"
+     any(ml.nlu_classifier(body.current_thread.text).intents,
+         .name == "cred_theft" and .confidence in~ ("medium", "high")
+     )
+     or any(attachments,
+            .file_type in $file_types_images
+            and any(file.explode(.),
+                    any(ml.nlu_classifier(.scan.ocr.raw).intents,
+                        .name == "cred_theft"
+                        and .confidence in ("medium", "high")
+                    )
+            )
+     )
+     or (
+       any(ml.nlu_classifier(body.current_thread.text).entities,
+           .name == "urgency"
+       )
+       and not any(ml.nlu_classifier(body.current_thread.text).intents,
+                   .name == "benign" and .confidence == "high"
+       )
```

Comment:
this one feels a little risky, but  trust your judgement if test rules looks good for a couple days

Reply:
This is what's currently in the rule :/  I'll try ripping it out and seeing how it goes!